docs/DafnyRef/Types.md

```diff
@@ -2915,14 +2916,6 @@ In contrast to one-dimensional arrays, there is no operation to
 convert stretches of elements from a multi-dimensional array to a
 sequence.
 
-## 15.3. Converting arrays to sequences {#sec-array-to-seq}
-
-To convert an array `x` to a sequence, use the `[..]` operator:
-```dafny
-  var intArr: array<int> := new int[] [1, 2, 3];
-  var intSeq: seq<int> := intArr[..];
-  assert intSeq == [1, 2, 3];
-```
```

Comment:
Why is this part gone?

Reply:
Duplicated in text at the end of 15.1, which is now the new 15.2